src/lib.rs

```diff
@@ -204,11 +204,14 @@ mod codegen;
 mod parser;
 mod value;
 mod error;
+mod util;
 
 pub mod short;
 pub mod object;
+pub mod number;
 
 use short::Short;
+use number::Number;
```

Comment:
Maybe we should `pub use` here to re-export it? Although it seems that it'll rarely need to be used, so may not be worth it and little in the way of usability gains: `use json::number::Number` vs. `use json::Number`; however the second variant does bring similarity with `JsonValue`s.